ui/contexts/alertActionHandler.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {
+  ReactElement,
+  createContext,
+  useCallback,
+  useContext,
+} from 'react';
+
+type AlertActionHandlerContextType = {
+  processAction: (actionKey: string) => void;
+};
+
+export const AlertActionHandlerContext = createContext<
+  AlertActionHandlerContextType | undefined
+>(undefined);
+
+export const AlertActionHandlerProvider: React.FC<{
+  children: ReactElement;
+}> = ({ children }) => {
+  // eslint-disable-next-line no-empty-function
+  const processAction = useCallback((_actionKey: string) => {}, []);
```

Comment:
Can we move this empty function to the usage within `confirm` rather than removing it here so we still have a functional context?